runtime/onert/backend/train/KernelGenerator.cc

```diff
@@ -370,7 +370,7 @@ void KernelGenerator::visit(const ir::train::operation::Pad &node)
   auto out_back_prop_tensor = _tensor_reg->getBackPropTensor(output_index);
   auto in_back_prop_tensor = _tensor_reg->getBackPropTensor(input_index);
 
-  fn->configure(input, output, pad, value, in_back_prop_tensor, out_back_prop_tensor);
+  fn->configure(input, pad, value, output, in_back_prop_tensor, out_back_prop_tensor);
```

Comment:
(For the reference)
https://github.com/Samsung/ONE/blob/ce19f6dd78cff2270be31aa91ac923a30ccbd88e/runtime/onert/backend/train/ops/PadLayer.cc#L46-L48